compiler/importer.nim

```diff
@@ -345,6 +351,7 @@ proc evalFrom*(c: PContext, n: PNode): PNode =
       if n[i].kind != nkNilLit:
         importSymbol(c, n[i], m, im.imported)
     c.addImport im
+    afterImport(c, m)
```

Comment:
so that re-exports are properly handled with imports like:
`from a import b`

Reply:
The next time, add your remarks as comments to the code.

Reply:
i usually distinguish between comments that only make sense during code review (to explain a diff) vs comments that explain code; here it could be argued either way